repository-data/webfiles/src/main/resources/site/freemarker/common/blog-article.ftl

```diff
@@ -4,7 +4,7 @@
 
 <#include "../include/imports.ftl">
 <#include "macro/sections/sections.ftl">
-<#include "macro/sectionNav.ftl">
+<#include "macro/stickyNavSections.ftl">
```

Comment:
We can remove this macro from blog-article.ftl, as Blog article does not have sticky nav.

Reply:
Good spot